lib/Jit/LLILCJit.cpp

```diff
@@ -303,6 +283,16 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
 
       // Tell the CLR that we've successfully generated code for this method.
       Result = CORJIT_OK;
+
+      if (Context.Options->DumpLevel >= ::DumpLevel::SUMMARY) {
+         errs() << "LLILC compiled: " 
+                << ", Entry = " << *NativeEntry 
+                << ", End = " << (*NativeEntry + *NativeSizeOfCode)
+                << ", size = " << *NativeSizeOfCode 
+                << " method = " << Context.MethodName 
+                << '\n';
+      }
+
```

Comment:
The reason to keep on a single line, and with the address first is so that the file can be sorted
and used as a map from addresses to method name.
When debugging I often want to know if a location is in code we jitted and this
give that capability.

I have a fix for the llilc_checkpass.py file that I'll be pushing as soon as I've tested it.
By using a non-capturing group in the regular expression I'll be able to avoid having
to renumber the captured groups.